src/components/ScholarshipForm.jsx

```diff
@@ -304,8 +309,11 @@ function ScholarshipForm({ scholarship, onSubmit }) {
 }
 
 ScholarshipForm.propTypes = {
-  onSubmit: PropTypes.func.isRequired,
   scholarship: PropTypes.object.isRequired,
+  status: PropTypes.string,
```

Comment:
Instead of `status` you can determine whether this is an edit by whether `scholarship.id` is present.

Reply:
But then if someone is viewing a scholarship, it would have `scholarship.id`